packages/db-migrations/databases/fxa/patches/patch-129-128.sql

```diff
@@ -0,0 +1,5 @@
+-- -- Delete new row from table
+-- DELETE FROM securityEventNames WHERE name='account.enable` OR name='account.disable;
```

Comment:
Missing ' after account.disable

Reply:
<code>WHERE name='account.enable`</code> looks like it might have the wrong closing quote too (backtick vs single quote).

Reply:
Great catches, thank you!